sign-in.txt

```diff
@@ -8,5 +8,6 @@
   Brian Bansenauer  |  Bansenauer-Cascadia
   Roger L           |  Floydfix
   Gori Jesionowski  |  gjesionowski
-  Edmund Chan       |  edmundchan8
+  Edmund Chan       |   edmundchan8
+  Richard Tae       |  taerichard
```

Comment:
Please put you name at the end of the list, rather than in the middle.

Reply:
You will need to pull develop and merge it into your code to see the additional name(s)

Reply:
Richard, 
The current develop branch has this list
﻿﻿//TODO: Be sure you have pulled develop and merged it into your feature branch 
        (created a feature branch with your name from develop if you don't have one yet)
//TODO: Edit this document to put your name and github username below the last name on the list below ... 
        ... then commit, push your feature branch to GitHub, and create (or review) your pull request
        
       NAME         |   GitHub USERNAME
  ________________  |  ___________________
  Brian Bansenauer  |  Bansenauer-Cascadia
  Roger L           |  Floydfix
  Gori Jesionowski  |  gjesionowski
  Edmund Chan       |  edmundchan8
  Nathanael Rorberg |  NathanaelRorberg
  Alain N           |  AlainNshimirimana